generators/server/templates/couchbase/src/test/java/package/config/CouchbaseTestContainer.java.ejs

```diff
@@ -35,9 +35,9 @@ public class CouchbaseTestContainer implements InitializingBean, DisposableBean
 
     @Override
     public void destroy() {
-        if (null != couchbaseContainer && couchbaseContainer.isRunning()) {
-            couchbaseContainer.stop();
-        }
+//        if (null != couchbaseContainer && couchbaseContainer.isRunning()) {
```

Comment:
Not used as bean isn't registered in the disposeable registry: should introduce a configuration bit as described [here](https://github.com/jhipster/generator-jhipster/issues/18757)